react-native-libraries.json

```
@@ -9204,5 +9204,12 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/inferusvv/react-native-unicorn-modals",
```

```suggestion
    "githubUrl": "https://github.com/inferusvv/react-native-unicorn-modals",
    "examples": [
      "https://snack.expo.dev/@inferusvv/react-native-unicorn-modals",
      "https://github.com/inferusvv/react-native-unicorn-modals/tree/main/example"
    ],
    "images": [
      "https://raw.githubusercontent.com/inferusvv/react-native-unicorn-modals/main/.github/images/demo.gif"
    ],
```
Let's add example apps and demo image.